<launch>

<!--  <node pkg="topic_tools" type="throttle" name="throttle_joint_states" args="messages joint_states 0.5" /> -->

  <node pkg="sbpl_two_arm_planner_node" type="requestPlan" name="request_plan" output="screen" respawn="false" >

    <param name="setup_experiment" value="true" />

    <param name="start_x" value="0.9" />  <!-- Works with 0.61 when grippers are flat-->
    <param name="start_y" value="0.00" />
    <param name="start_z" value="0.95" />
    <param name="start_roll" value="0.0" />
    <param name="start_pitch" value="0.0" />
    <param name="start_yaw" value="0.0" />

    <param name="goal_x" value="0.9" />
    <param name="goal_y" value="0.0" />
    <param name="goal_z" value="1.32" /> <!-- 1.28 -->
    <param name="goal_roll" value="0.0" />
    <param name="goal_pitch" value="0.0" />
    <param name="goal_yaw" value="0.0" />

    <param name="object_offset_x" value="0.82" />
    <param name="object_offset_y" value="-0.0" />
    <param name="object_offset_z" value="0.07" />

    <param name="raise_torso" value="true" />
    <param name="go_to_start" value="true" />
    <param name="plan_path" value="true" />
    <param name="execute_plan" value="true" />
    <param name="filter_trajectory" value="true" />

    <param name="attached_object_filename" value="$(find sbpl_two_arm_planner_node)/tray2.txt" />
    <param name="known_objects_filename" value="$(find sbpl_two_arm_planner_node)/env/bookshelf2.env" />
    <param name="remove_current_object" value="pole" />

    <param name="right_arm_pose_on_object_x" value="-0.14" />
    <param name="right_arm_pose_on_object_y" value="-0.12" />
    <param name="right_arm_pose_on_object_z" value="0.0" />
    <param name="right_arm_pose_on_object_roll" value="1.57" />
    <param name="right_arm_pose_on_object_pitch" value="0.0" />
    <param name="right_arm_pose_on_object_yaw" value="0.0" />

    <param name="left_arm_pose_on_object_x" value="-0.14" />
    <param name="left_arm_pose_on_object_y" value="0.12" />
    <param name="left_arm_pose_on_object_z" value="0.00" />
    <param name="left_arm_pose_on_object_roll" value="1.57" />
    <param name="left_arm_pose_on_object_pitch" value="0.00" />
    <param name="left_arm_pose_on_object_yaw" value="0.00" />

  </node>

</launch>
